flutter/lib/src/jvm/jvm_exception.dart

```diff
@@ -0,0 +1,113 @@
+import 'jvm_frame.dart';
+
+const _emptyString = '';
+const _causedBy = 'Caused by:';
+const _suppressed = 'Suppressed:';
+const _newLine = '\n';
+
+class JvmException {
```

Comment:
Let's annotate with @internal, all of them, what do you think?

Reply:
It's not exposed, so I think the user will get a warning anyway if he uses it.